actionview/lib/action_view/railtie.rb

```diff
@@ -37,7 +37,7 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
-    initializer "action_view.collection_caching" do |app|
+    initializer "action_view.collection_caching", after: "action_controller.set_configs" do |app|
```

Comment:
Brought to my attention that we don't need the `on_load` when we're using `:after`. Killed it in https://github.com/rails/rails/commit/940124a22676f51540402e538fff626ef73b1d99.